src/lib/y2storage/proposal_settings.rb

```diff
@@ -50,6 +50,10 @@ def initialize
       @home_filesystem_type     = ::Storage::FsType_XFS
       @enlarge_swap_for_suspend = false
     end
+
+    def use_encryption
```

Comment:
This should be `use_encryption?` (with question mark). Although I would prefer `encrypt?` for consistency with the equivalent methods already present in `LvmHelper` and `PlannedVolume`